### PR TITLE
Devuploads UP

### PR DESCRIPTION
**Pull request by BlazeFTL**
`https://devuploads.com/8xf23hv312bd`

Site Owner Gave Up😬🤡

### CODE REVIEW

File: filters/filters-2022.txt [hunk 1]
```diff
@@ -4670,6 +4667,17 @@ miuiflash.com##+js(acs, document.createElement, adb)
 @@||adoto.net/dashboard/common/js/common.js$3p,script,domain=dev.miuiflash.com|djxmaza.in|thecubexguide.com
 dev.miuiflash.com,djxmaza.in,thecubexguide.com##+js(no-fetch-if, /adtrue\.com|eningspon\.com|freychang\.fun|orquideassp\.com|popunder/)
 dev.miuiflash.com,djxmaza.in,thecubexguide.com##+js(no-xhr-if, /a-mo\.net|adnxs\.com|prebid|creativecdn\.com|e-planning\.net|quantumdex\.io/)
+dev.miuiflash.com,djxmaza.in,thecubexguide.com##iframe:upward(div)
```
[the remark this answers is not in the document]
Comment (JobcenterTycoon) [reply filters/filters-2022.txt:4670]: But not by Firefox :( I heard `:upward()` is better than `:has()` as long native `:has()` is not supported.

**Comment by JobcenterTycoon** [comment]
I get anti adblock with this but no anti adblock with the current filter (tested on FF)

File: filters/filters-2022.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (JobcenterTycoon) [reply filters/filters-2022.txt:4670]: 1.19 is latest stable so it will still need weeks until native `:has()` will be avaiable.

**Comment by BlazeFTL** [comment]
> I get anti adblock

So Do I🤣...Was Working Before I'll Fix Later(For Myself)..I Dont Like That Transform Thingy😅

**Comment by BlazeFTL** [comment]
@JobcenterTycoon Bro Last Try If Still Doesn't Work Close This PR😅(Sry For The Wasted Time)

**Comment by JobcenterTycoon** [comment]
Works fine now. Good work!

**Pull request merged**